openedx/frontend-app-authoring
# Fix: some default setting not updating with updated default settings

## Description

Some default setting ( **Show reset option**, **Show answer**, **Randomization**) not updating with updated default settings.

## Testing instructions
1. Check in **Studio** -> **Settings** -> **Advanced settings** values for `Show reset button for problems`, `Show answer` and `Randomization`:
![212](https://github.com/user-attachments/assets/e4638f96-d2ec-4c3f-bcbe-60b751c09940)
2. Add problem -> check values for options - verify that they are the same as in **Advanced settings**
![213](https://github.com/user-attachments/assets/917c6e77-c797-497d-af3e-cf5e9180d9eb)
3. For problem change values for same options to some other values (not default), save changes
![214](https://github.com/user-attachments/assets/e3e8f4ae-1d36-4db8-b4b0-e9a6239df609)
4. Click on Edit for problem and set for options same values as the default ones in **Advanced settings** (as they were in step 2), save changes
5. Check if new values were set

### Actual Result: 
Previous values are displayed (from step 3) - not as they were set in step 4
